src/content/blog/2024_mitiq_impact.md

```diff
@@ -0,0 +1,147 @@
+---
+title: The impact of Mitiq
+author: nate stemen
+day: 24
+month: 12
+year: 2024
+tags: 
+  - python
+  - mitiq
+  - community
+---
+
+As we approach the 5 year anniversary since the Mitiq project has begun, we're starting to see how the project has played a role in shaping the ecosystem.
+Mitiq was created in early 2020 with the intention to *make error mitigation easily available to everyone running programs on quantum computers.*
+Have we acheived this goal?
+Making QEM easily available to **everyone** is an ambitious target, but as the adoption metrics below demonstrate, Mitiq has significantly lowered the barrier to using Quantum Error Mitigation (QEM) protocols.
+
+As the field of Quantum Error Mitigation matured, Mitiq has gained recognition as an optional, but important component of the quantum computing stack.
+While some skeptics argue that Quantum Error Correction (QEC) will eventually render QEM obsolete, others, like John Preskill, believe that the two approaches will complement one another even in the long term.
+In his [talk at Q2B 2024](https://preskill.caltech.edu/talks/Preskill-Q2B-2024.pdf), Preskill stated:
+
+> <p style="font-size: 24px;">Error mitigation will continue to be useful in the Megaquop era and beyond.</p>
+
+Preskill’s remarks underscore the enduring importance of error mitigation and highlight the need for an open-source, platform-agnostic tool like Mitiq to support the growing quantum ecosystem.
+When Mitiq was first introduced in 2020, the landscape of QEM tools was nascent, with early efforts like pyquil’s measurement error mitigation and Qiskit Ignis paving the way.
+Since then, the ecosystem has expanded, with Mitiq leading the charge by implementing nine QEM techniques and offering official support for 6 quantum SDKs (and un-official support for more).
+
+Let’s take a closer look at how Mitiq has shaped the field and grown over time.
+
+
+## Adoption and Community Growth
+
+At the core of making QEM available to all, is building a platform-agnostic tool.
+Whether you're writing your quantum program with Qiskit, Cirq, PyQuil, Qibo, Braket, ..., you can use Mitiq!
+This has been a major driving force behind the growth and adoption of the project, and allows for users to write QEM code once with the flexibility of running on any (gate-based) hardware.
+As the only SDK-agnostic QEM tool[^2], Mitiq’s adoption metrics provide a clear measure of its utility and impact.
+
+[^2]: To create such a tool in the ever-changing quantum software landscape that does not have an established IR, is, to say the least, a lot of work. In particular the **maintenance** overhead required to stay up to date with $n$ SDKs is quadratic.
+
+- **Downloads:** Over 195,000 downloads (and counting!).
```

Comment:
it should be clear from the graph below, but it's still better to specify this is _total_ since inception, and not /year